crates/bevy_animation/src/lib.rs

```diff
@@ -294,6 +406,44 @@ pub fn animation_player(
     }
 }
 
+fn _find_entity(
+    entity: Entity,
+    path: &EntityPath,
+    children: &Query<&Children>,
+    names: &Query<&Name>,
+) -> Option<Entity> {
+    // PERF: finding the target entity can be optimised
+    let mut current_entity = entity;
+    // Ignore the first name, it is the root node which we already have
+    for part in path.parts.iter().skip(1) {
+        let mut found = false;
+        if let Ok(children) = children.get(current_entity) {
+            for child in children.deref() {
+                if let Ok(name) = names.get(*child) {
+                    if name == part {
+                        // Found a children with the right name, continue to the next part
+                        current_entity = *child;
+                        found = true;
+                        break;
+                    }
+                }
+            }
+        }
+        if !found {
+            warn!("Entity not found for path {:?} on part {:?}", path, part);
+            return None;
+        }
+    }
+    Some(current_entity)
+}
+
+fn _update_transitions(player: &mut AnimationPlayer, time: &Time) {
```

Comment:
```suggestion
fn update_transitions(player: &mut AnimationPlayer, time: &Time) {
```

We don't need to prefix our private module functions with `_`.